lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp

```diff
@@ -242,6 +242,62 @@ static Value createTMTensorScanOp(
   return scanOp->getResult(0);
 }
 
+// Utility function to create a TMTensor::SortOp. Returns `true` if success,
+// `failure` otherwise.
+bool createTMTensorSortOp(PatternRewriter *rewriter, const Location &sortOploc,
+                          const llvm::ArrayRef<Value> &operands,
+                          const llvm::ArrayRef<Type> &elementTypes,
+                          int64_t dimension, bool isStable, bool isDescending,
+                          TMTensor::SortOp &sortOp) {
```

Comment:
The convention here is to pass `PatternRewriter` as a reference, and all the others by value.